apps/chat-e2e/src/ui/selectors/sideBarSelectors.ts

```diff
@@ -37,7 +37,7 @@ export const PromptBarSelectors = {
   newFolder: '[data-qa="create-prompt-folder"]',
   promptFolders: '[data-qa="prompt-folders"]',
   newPromptButton: '[data-qa="new-prompt"]',
-  prompts: '[data-qa="prompts"]',
+  prompts: '[data-qa="promps-section-container"] >> [data-qa="prompts"]',
```

Comment:
could you please fix typo 'promps' in `apps/chat/src/components/Promptbar/components/Prompts.tsx`

Reply:
It will not work, it should be changed on the UI-side

Reply:
![image](https://github.com/epam/ai-dial-chat/assets/81012703/9c01ebcb-7dfa-4d05-a760-039e14a205ce)

Reply:
fixed